src/modules/ToDoList.js

```diff
@@ -0,0 +1,50 @@
+export default class ToDoList {
+  constructor(taskList = []) {
+    this.taskList = taskList;
+  }
+
+  get getTaskList() {
+    return this.taskList;
+  }
+
+  set setTaskList(newTaskList = []) {
+    if (!newTaskList) return;
+    this.taskList = newTaskList;
+  }
+
+  getLength() {
+    return this.taskList.length;
+  }
+
+  addTask(task) {
+    this.taskList.push(task);
+  }
+
+  removeTask(index) {
+    this.taskList.splice(index, 1);
+    this.updateTaskIndex();
+  }
+
+  updateTaskIndex() {
+    this.taskList.sort((a, b) => a.index - b.index);
+    for (let i = 0; i < this.taskList.length; i += 1) {
+      this.taskList[i].index = i;
+    }
+  }
+
+  removeCompleted() {
+    this.taskList = this.taskList.filter((task) => !task.completed);
+  }
+
+  sortTasks() {
+    for (let i = 0; i < this.taskList.length - 1; i += 1) {
+      for (let j = 0; j < (this.taskList.length - 1) - i; j += 1) {
+        if (this.taskList[j].index > this.taskList[j + 1].index) {
+          const temp = this.taskList[j + 1];
+          this.taskList[j + 1] = this.taskList[j];
+          this.taskList[j] = temp;
+        }
+      }
+    }
+  }
```

Comment:
It would be good that in order to make your code more readable, you can replace `for` loops with `this.taskList.forEach` as long as `taskList` is an Array and `forEach` is a method available to iterate through Array elements. Please follow the link of the [DRY, KISS and YAGNI lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md) and review the **KISS** material.

Reply:
The changes that you suggested here did my code it was seen better, thank you @luigirazum! 💯 😃 
Here below l sent an image about the change. 👇 
![Captura de pantalla (293)](https://user-images.githubusercontent.com/79432268/208811838-5178cdc0-3450-41fc-aa57-df55801f5a23.png)